content/en/real_user_monitoring/browser/modifying_data_and_context.md

```diff
@@ -345,7 +345,11 @@ The following attributes are optional but Datadog recommends providing at least
 
 Increase your filtering capabilities by adding extra attributes on top of the recommended ones. For instance, add information about the user plan, or which user group they belong to.
 
-To identify user sessions, use the `setUser` API:
+When making changes to the user session object, all RUM events collected afterwards the change contain the updated information.
+
+## User session APIs
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about having a sub section for each capability (like global context)?
something like:
```
## user sessions
### identify user session
### access user session
### override user session property
### clear user session
```

Reply:
it could be nice to merge 
```
## Identify user sessions
## User session APIs
```
into
```
## User session
```

it regroup all things related to user session in a single section, like Global Context